server/src/config/config.js

```diff
@@ -14,12 +14,18 @@ module.exports = {
     database: DEVDATABASE,
     host: "localhost",
     dialect: "postgres",
+    logging: false,
```

Comment:
why set logging to false?

Reply:
I got tired of the queries logging on my terminal 😒

Reply:
lmao, it's good for debugging in dev environment. just remove for production environments